loader/core/src/main/java/pl/skidam/automodpack_loader_core/client/ModpackUtils.java

```diff
@@ -51,18 +51,8 @@ public static boolean isUpdate(Jsons.ModpackContentFields serverModpackContent,
                     if (modpackContentField.editable) continue;
                 } else {
                     Path standardPath = CustomFileUtils.getPathFromCWD(file);
-                    if (Files.exists(standardPath) && Objects.equals(serverSHA1, CustomFileUtils.getHash(standardPath))) {
-                        LOGGER.info("File {} already exists on client, coping to modpack", file);
-                        try {
-                            CustomFileUtils.copyFile(standardPath, path);
-                        } catch (IOException e) {
-                            e.printStackTrace();
-                        }
-                        continue;
-                    } else {
-                        LOGGER.info("File does not exists {} - {}", standardPath, file);
-                        return true;
-                    }
+                    LOGGER.info("File does not exists {} - {}", standardPath, file);
```

Comment:
While we're here: There's a little grammar error.
```suggestion
                    LOGGER.info("File does not exist {} - {}", standardPath, file);
```